src/components/utils/aggrid/custom-set-filter.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, {
+    forwardRef,
+    useEffect,
+    useImperativeHandle,
+    useState,
+} from 'react';
+import { IDoesFilterPassParams, IFilterParams } from 'ag-grid-community';
+import { Option } from 'components/results/shortcircuit/shortcircuit-analysis-result.type';
+
+// TODO temporary, remove it when SCA results will use full custom headers and filters
+export default forwardRef((props: IFilterParams, ref) => {
```

Comment:
Give this component a default name ? CustomSetFilter ?

Reply:
Ok done